scene/gui/tree.cpp

```diff
@@ -4439,6 +4439,14 @@ void Tree::item_edited(int p_column, TreeItem *p_item, MouseButton p_custom_mous
 	if (p_item != nullptr && p_column >= 0 && p_column < p_item->cells.size()) {
 		edited_item->cells.write[p_column].dirty = true;
 	}
+	if (blocked > 0) {
+		callable_mp(this, &Tree::emit_item_edited).call_deferred(p_custom_mouse_index);
```

Comment:
This either needs a comment, because being blocked does not prevent firing signal, or it should be solved in the problematic code by connecting with `CONNECT_DEFERRED`.

Reply:
Poke.

Reply:
Ping @bruvzg :)

Reply:
`CONNECT_DEFERRED` might be enough, will check it a moment.